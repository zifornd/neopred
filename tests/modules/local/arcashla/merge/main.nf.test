nextflow_process {

    name "Test Process ARCASHLA_MERGE"
    script "modules/local/arcashla/merge/main.nf"
    process "ARCASHLA_MERGE"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                // define inputs of the process here. Example:
                // input[0] = file("test-file.txt")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
