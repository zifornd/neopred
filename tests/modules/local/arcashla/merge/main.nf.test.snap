{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "genotypes.tsv:md5,f57dbda547fc6e900a0f8a585ddd9b49"
                ],
                "1": [
                    "versions.yml:md5,747d81dd1a0fc8cc8b7139887feeb80b"
                ],
                "merged_gt": [
                    "genotypes.tsv:md5,f57dbda547fc6e900a0f8a585ddd9b49"
                ],
                "versions": [
                    "versions.yml:md5,747d81dd1a0fc8cc8b7139887feeb80b"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.1"
        },
        "timestamp": "2024-12-09T06:33:52.869312"
    }
}